bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.13.3
+version: 12.13.6
```

There is a typo bumping the chart version
```suggestion
version: 12.13.4
```